packages/SideNavigation/SubMenu/__tests__/__snapshots__/SubMenu.spec.jsx.snap

```diff
@@ -1,23 +1,20 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`SideNavigation.SubMenu renders 1`] = `
-<div
-  className="mainDiv"
->
-  <button
-    className="buttonSubMenu active"
+<div>
+  <styled.button
```

Comment:
Could you please change `shallow` rendering to `mount` for snapshots so that it will include css generated by `styled-components`.

Reply:
Done